src/cpu/pred/BranchPredictor.py

```diff
@@ -63,14 +178,23 @@ class BranchPredictor(SimObject):
     abstract = True
 
     numThreads = Param.Unsigned(Parent.numThreads, "Number of threads")
-    BTBEntries = Param.Unsigned(4096, "Number of BTB entries")
```

Comment:
This will affect quite a few configuration files in `configs/common/cores/` and `src/python/gem5/prebuilt`. A quick `git grep` should find them.

Reply:
I updated all config files where I could find the BTB parameters